Packs/CommonScripts/Scripts/ParseHTMLIndicators/ParseHTMLIndicators.yml

```diff
@@ -0,0 +1,35 @@
+commonfields:
+  id: ParseHTMLIndicators
+  version: -1
+name: ParseHTMLIndicators
+script: ''
+type: python
+tags: []
+comment: This script extracts indicators from HTML and handles bad top-level domains to avoid false positives caused by file extensions.
+enabled: true
+args:
+- name: url
+  required: true
+  description: The full URL of the blog.
+- name: excludeIndicators
```

Comment:
arguments should follow the snake_case convention

Reply:
Done.